content/en/seavoice/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title : "SeaVoice"
-description: "SeaVoice demos and tutorials for TTS and STT"
+title : "SeaVoice Humanlike Text to Speech & Speech to Text with Cutting Edge Accuracy"
```

Comment:
instead of repeating "cutting edge", maybe "State-of-the-art"?

Reply:
I chose cutting-edge because surfer says it has a high search volume (49,500) than state-of-the-art. The top query seems to be "state of art", and many people are even searching "state of art meaning". I guess cutting edge is more popular among the general mass?